### DISCUSSION TITLE
GCODE over API

### DISCUSSION THREAD

**Discussion by dlucksinger** [luc-github/ESP3D#1026] [Questions]
Hello,

I would like to send GCODE commands over the API / Websocket interface via a json string.  I was looking at the features and didn't really see this in there.  Can someone help me with this?

**Comment by luc-github** [comment]
the web socket communication is available in esp3d 3.0
https://esp3d.io/esp3d/v3.x/documentation/api/websockets/index.html

but it does not use json just directly forward GCODE

**Reply by dlucksinger** [reply]
Thank you, I just don't know how I would send code via the program I'm using.  It can send a JSON string to a api, that's what I'm shooting for.

**Reply by luc-github** [reply]
> I just don't know how I would send code via the program I'm using.

what program ?

**Comment by dlucksinger** [comment]
OBS, SAMMI, Streamber bot
Looking to allow chat to move gantry
    On Sunday, May 26, 2024 at 11:51:17 p.m. MDT, Luc ***@***.***> wrote:  
 
 



I just don't know how I would send code via the program I'm using.


what program ?

**Reply by luc-github** [reply]
ho ok , sorry, no idea of what it is, so cannot help on this

**Comment by dlucksinger** [comment]
No worries thank you. 
    On Tuesday, May 28, 2024 at 09:56:57 p.m. MDT, Luc ***@***.***> wrote:  
 
 


ho ok , sorry, no idea of what it is, so cannot help on this
